plugins/lookup/onepassword.py

```diff
@@ -462,10 +462,10 @@ def _parse_field(self, data_json, field_name, section_title=None):
                 # If the field name doesn't exist in the section, match on the value of "label"
                 # then "id" and return "value"
                 if field.get("label") == field_name:
-                    return field["value"]
+                    return field.get("value")
```

Comment:
Use `""` for the default value.
```suggestion
                    return field.get("value", "")
```